LBH.AdultSocialCare.Api/V1/Profiles/MappingProfile.cs

```diff
@@ -109,13 +111,26 @@ public MappingProfile()
             CreateMap<CareChargeReclaimBulkUpdateRequest, CareChargeReclaimBulkUpdateDomain>();
             CreateMap<CareChargeReclaimUpdateRequest, CarePackageReclaimUpdateDomain>();
 
-            #endregion
+            #endregion CarePackageReclaim
 
             #region PayRun
 
             CreateMap<Payrun, DraftPayRunCreationDomain>().ReverseMap();
 
-            #endregion
+            #endregion PayRun
+
+            #region Invoice item
+
+            CreateMap<PayRunInvoiceItemDomain, PayRunInvoiceItemResponse>()
+                .ForMember(response =>
```

Comment:
seems to be a merging issue - this one has been removed

Reply:
Should I delete that section?

Reply:
Believe I should. Done